### PR TITLE
add addon-manager owa details

### PR DESCRIPTION
**Pull request by patlub**
The additions made will enable the addon manager owa to be listed in the addon index.

### CODE REVIEW

File: src/main/resources/add-ons-to-index.json [hunk 1]
```diff
@@ -43,6 +43,27 @@
       }
     },
     {
+    "uid": "org.openmrs.owa.addonmanager",
+    "type": "OWA",
+    "name": "Add-On-Manager",
```
Comment (dkayiwa) [inline src/main/resources/add-ons-to-index.json]: Can you have the name without hyphens?

File: src/main/resources/add-ons-to-index.json [hunk 2]
```diff
@@ -43,6 +43,27 @@
...
+      {
+        "name": "OpenMRS",
+        "url": "https://github.com/openmrs"
+      }
+    ],
+    "backend": "org.openmrs.addonindex.backend.Bintray",
+    "bintrayPackageDetails": {
+      "owner": "openmrs",
+      "repo": "owa",
+      "package": "Add-On-Manager"
```
Comment (dkayiwa) [inline src/main/resources/add-ons-to-index.json]: Can you also change the package name to be in the same format as that for other OWAs?

File: src/main/resources/add-ons-to-index.json (hunk 1 above)
Comment (patlub) [reply src/main/resources/add-ons-to-index.json]: Done !

File: src/main/resources/add-ons-to-index.json (hunk 2 above)
Comment (patlub) [reply src/main/resources/add-ons-to-index.json]: I've made the change

File: src/main/resources/add-ons-to-index.json [hunk 3]
```diff
@@ -43,6 +43,27 @@
       }
     },
     {
+    "uid": "org.openmrs.owa.addonmanager",
+    "type": "OWA",
+    "name": "Add On Manager OWA",
```
Comment (dkayiwa) [inline src/main/resources/add-ons-to-index.json]: Drop OWA from the name

File: src/main/resources/add-ons-to-index.json [hunk 4]
```diff
@@ -43,6 +43,27 @@
...
+      {
+        "name": "OpenMRS",
+        "url": "https://github.com/openmrs"
+      }
+    ],
+    "backend": "org.openmrs.addonindex.backend.Bintray",
+    "bintrayPackageDetails": {
+      "owner": "openmrs",
+      "repo": "owa",
+      "package": "openmrs-addOnManager"
```
Comment (dkayiwa) [inline src/main/resources/add-ons-to-index.json]: Have you looked at the formate of the package for other OWAs?

File: src/main/resources/add-ons-to-index.json (hunk 3 above)
Comment (patlub) [reply src/main/resources/add-ons-to-index.json]: Fixed !

File: src/main/resources/add-ons-to-index.json (hunk 4 above)
Comment (patlub) [reply src/main/resources/add-ons-to-index.json]: Its fixed

Comment (dkayiwa) [reply src/main/resources/add-ons-to-index.json]: Are you done with the other ticket of changing the addon display name and download zip file?

Comment (patlub) [reply src/main/resources/add-ons-to-index.json]: Yes am done.

**Pull request merged**